packaging/ansible-runner-service-project/project/ovirt-host-upgrade.yml

```diff
@@ -35,6 +35,7 @@
           - 'ca': "{{ ovirt_vdsm_trust_store }}/{{ ovirt_vdsm_spice_ca_file }}"
             'cert': "{{ ovirt_vdsm_trust_store }}/{{ ovirt_vdsm_spice_cert_file }}"
         register: iscorrect
+        ignore_errors: yes
```

Comment:
why?

Reply:
Because if the certificate is going to expire within the given period, openssl will return a non-zero code and the playbook stops here with an error.

Reply:
so maybe we should add a comment there?

Reply:
+1

and what about other certificates? vmconsole? ovn?

Reply:
I cannot see another use of openssl verify in the ansible stuff.

The comment added.

Please note that there are other issues with vmconsole certificates, which will be handled in a future, separate PR.